ts/features/idpay/onboarding/screens/MultiValuePrerequisitesScreen.tsx

```diff
@@ -0,0 +1,145 @@
+import IconFont from "../../../../components/ui/IconFont";
+import I18n from "../../../../i18n";
+import { IOStackNavigationProp } from "../../../../navigation/params/AppParamsList";
+import { IDPayOnboardingParamsList } from "../navigation/navigator";
+import { useMultiPrerequisitesPagination } from "../utils/hooks";
+import { useOnboardingMachineService } from "../xstate/provider";
+
+const styles = StyleSheet.create({
+  maxheight: {
+    height: "100%"
+  }
+});
+
+type ListItemProps = {
+  text: string;
+  checked: boolean;
+  onPress: () => void;
+};
+
+const CustomListItem = ({ text, onPress, checked }: ListItemProps) => (
+  <NBListItem
+    style={{
+      flexDirection: "row",
+      justifyContent: "space-between",
+      paddingVertical: 12
+    }}
+    onPress={onPress}
+  >
+    <H4 weight={checked ? "SemiBold" : "Regular"} color={"bluegreyDark"}>
+      {text}
+    </H4>
+    <IconFont
+      name={checked ? "io-radio-on" : "io-radio-off"}
+      size={22}
+      color={checked ? IOColors.blue : IOColors.bluegrey}
+    />
+  </NBListItem>
+);
+
+const buttonProps = {
+  leftButton: { title: I18n.t("global.buttons.back"), bordered: true },
+  rightButton: {
+    title: I18n.t("global.buttons.continue"),
+    bordered: false
+  }
+};
+
+type MultiValueScreenNavigationType = IOStackNavigationProp<
+  IDPayOnboardingParamsList,
+  "IDPAY_ONBOARDING_MULTI_SELF_DECLARATIONS"
+>;
+type MultiValuePrerequisitesScreenRouteParams = {
+  page: number;
+};
+
+type MultiValuePrerequisitesScreenRouteProps = RouteProp<
+  IDPayOnboardingParamsList,
+  "IDPAY_ONBOARDING_MULTI_SELF_DECLARATIONS"
+>;
+
+type NavigationProps = {
+  navigation: MultiValueScreenNavigationType;
+};
+const MultiValuePrerequisitesScreen = ({ navigation }: NavigationProps) => {
+  const [selectedIndex, setSelectedIndex] = React.useState<number | undefined>(
+    undefined
+  );
+  const { params } = useRoute<MultiValuePrerequisitesScreenRouteProps>();
+  const machine = useOnboardingMachineService();
+
+  const { currentPage, confirmChoice, goBack } =
+    useMultiPrerequisitesPagination(navigation, machine, params.page);
+
+  const continueOnPress = () => {
+    if (selectedIndex === undefined) {
+      return null;
+    }
+    confirmChoice({
+      _type: currentPage._type,
+      value: currentPage.value[selectedIndex],
+      code: currentPage.code
+    });
+
+    return null;
+  };
+
+  return (
+    <SafeAreaView style={IOStyles.flex}>
+      <BaseScreenComponent
+        goBack={goBack}
+        headerTitle={I18n.t("idpay.onboarding.headerTitle")}
+      >
+        <View style={IOStyles.horizontalContentPadding}>
+          <H1>{I18n.t("idpay.onboarding.multiPrerequisites.header")}</H1>
+          <NBView spacer={true} />
+          <Body>{I18n.t("idpay.onboarding.multiPrerequisites.body")}</Body>
+          <Link>{I18n.t("idpay.onboarding.multiPrerequisites.link")}</Link>
+          <NBView spacer large />
```

Comment:
```suggestion
          <VSpacer size={16} />
```

Use the new `VSpacer` component